python/mxnet/onnx/mx2onnx/_export_onnx.py

```diff
@@ -91,15 +91,22 @@ def convert_layer(node, **kwargs):
 
         op = str(node["op"])
         opset_version = kwargs.get("opset_version", onnx_opset_version())
+        if opset_version < 12:
+            logging.warning('Your ONNX op set version is %s, '  % str(opset_version) +
+                            'which is lower than then lowest tested op set (12), please consider '
+                            'updating ONNX')
+            opset_version = 12
         # fallback to older opset versions if op is not registered in current version
+        convert_func = None
         for op_version in range(opset_version, 11, -1):
             if op_version not in MXNetGraph.registry_ or op not in MXNetGraph.registry_[op_version]:
-                if opset_version == 12:
-                    raise AttributeError("No conversion function registered for op type %s yet." % op)
                 continue
             convert_func = MXNetGraph.registry_[op_version][op]
             break
 
+        if convert_func is None:
+            raise AttributeError("No conversion function registered for op type %s yet." % op)
```

Comment:
can you add a comment here to explain why this would happen? (ie. is it because the op is not supported or because the installed version of onnx is too old?)

Reply:
added!